portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeErcForCategories.testcase

```diff
@@ -8,11 +8,9 @@ definition {
 	setUp {
 		TestCase.setUpPortalInstance();
 
-		User.firstLoginPG();
+		JSONDepot.addDepot(depotName = "Test Depot Name");
 
-		task ("Given an asset library is created") {
-			JSONDepot.addDepot(depotName = "Test Depot Name");
-		}
```

Comment:
Not sure why the Given was removed. Wouldn't it still make sense to wrap this with a Given description here?